roles/artemis/defaults/main.yml

```diff
@@ -100,6 +100,10 @@ enable_science_event_logging: false
 
 push_notification_relay: "https://hermes.artemis.cit.tum.de"
 
+sentry:
+  dsn:
+  enable_db_tracing: false
```

Comment:
Please add optional variables (`sentry.send_default_pii`, `sentry.traces_sample_rate`, and `sentry.environment`) as well as (mainly for documentation).

If they are optional, please comment them out.